### ISSUE TITLE
Legend of Tank: Endless attack when fighting "Battleship Mikasa" (戦艦三笠)

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Player#2154]
#### Name of the game:
Legend of Tank
https://www.freem.ne.jp/win/game/2356

#### Attach files (as a .zip archive or link them)
[LegendOfTank_Save03.zip](https://github.com/EasyRPG/Player/files/4478174/LegendOfTank_Save03.zip)

#### Describe the issue in detail and how to reproduce it:
Load the save, talk to Olaf (the one in the middle).

Fast forward through the battle, after a while you will encounter an infinite attack loop.
Such loops can also happen in normal gameplay: When all actors are unable to move and don't recover.

**Comment by Ghabry** [comment]
The execution order is correct but the movement check is wrong:
```
Enemy attacks (Enables Switch 100)
Page 4 executes -> Entire Party No Move
Page 2 executes
Page 2 executes
Enemy attacks
Page 4 executes
Page 2 executes
Enemy attacks
Page 4 executes
Page 2 executes
Page 3 executes -> Party can move
Turn ends in RPG_RT, but not in Player (Player runs "Page 4")
```

**Comment by fmatthew5876** [comment]
I can't reproduce this anymore in master.

I talk to the pirate guy in the middle and it starts a sea battle with a ship. I tried several times and it never gets stuck.

I do remember trying this a long time ago and getting the hang. So maybe something fixed it?

**Comment by Ghabry** [comment]
Was curious here, so made a bisect. This was fixed by @rueter37 in 44052955d0b377737575ace2da7d917022469e91
